content/en/docs/concepts/configuration/even-pods-spread.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Even Pods Spread
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+This page explains how EvenPodsSpread feature works and the mechanics under the hood.
```

Comment:
```suggestion
You can use _spread constraints_ to control how Pods are spread across your cluster among failure-domains such as regions, zones, nodes, and other user-defined topology domains. This can help to achieve high availability as well as efficient resource utilization.
```

Reply:
Also consider using a glossary tooltip for Pods: `{{< glossary_tooltip text="Pods" term_id="pod" >}}`

Reply:
It's more precise to use "topology spread constraints".